templates/admin/base_site.html

```diff
@@ -0,0 +1,435 @@
+    font-size: var(--font-size-peach);
+    padding: 4px 0 !important;
+  }
+
+  #changelist-filter h3::after {
+    content: ":";
+  }
+
+  #changelist-filter ul {
+    display: flex;
+    padding: 0;
+    border: none;
+    margin-bottom: 16px;
+  }
+
+  #changelist-filter li {
+    margin-left: 150px !important;
+    padding: 0 !important;
+  }
+
+  #changelist-filter li:first-child {
+    margin-left: 0 !important;
+  }
+
+  #changelist-filter li.selected {
+    border: none;
+    border-bottom: 1px solid var(--color-border);
+    font-weight: bold;
+  }
+
+  #changelist-filter li a:hover {
+    color: var(--color-blue) !important;
+  }
+
+  #content .addlink,
+  #content .changelink,
+  #content .viewlink {
+    background: none;
+    padding: 8px 12px;
+    border-radius: var(--border-radius-standard);
+    border: 1px solid var(--color-border) !important;
+  }
+
+  #content .addlink:hover,
+  #content .changelink:hover,
+  #content .viewlink:hover {
+    background-color: var(--color-blue) !important;
+  }
+
+  /* DASHBOARD */
+  .dashboard #container #content {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+  }
+
+  .dashboard #container h1 {
+    order: 0;
+  }
+
+  .dashboard #content-related {
+    order: 1;
+  }
+
+  .dashboard #content-main {
+    order: 2;
+  }
+
+  .dashboard #container th,
+  .dashboard #container caption {
+    padding: 0;
+    vertical-align: middle;
+  }
+
+  .dashboard #container caption {
+    border-bottom: 1px solid var(--color-border);
+    font-weight: bold;
+  }
+
+  .dashboard #container th,
+  .dashboard #container td {
+    border: none;
+  }
+
+  .dashboard #container #content #recent-actions-module h2 {
+    border-bottom: 1px solid var(--color-border);
+    margin-bottom: 16px;
+    padding: 0;
+  }
+
+  .dashboard #container #content #recent-actions-module h3 {
+    display: none;
+  }
+
+  .dashboard #content th a {
+    font-weight: bold;
+    transition: border 0.3s ease-in-out;
+  }
+
+  .dashboard #content th a:hover,
+  .dashboard #content caption a:hover {
+    border-bottom: none;
+    text-decoration: none;
+    color: var(--color-blue) !important;
+  }
+
+  /* SCHEMA DETAILS PAGE */
+  #schema_form {
+    padding: 0;
+  }
+
+  .field-json_schema #json-renderer {
+    font-family: var(--font-family-mono);
+    line-height: 32px;
+    padding: 0;
+    font-size: var(--font-size-peach);
+    background: black !important;
+  }
+
+  .field-json_schema #json-renderer * {
+    line-height: inherit;
+    font-size: inherit;
+    border-color: var(--color-border);
+  }
+
+  #json-renderer ul {
+    margin-left: 32px;
+  }
+
+  #json-renderer li {
+    font-weight: bold;
+  }
+
+  #json-renderer .json-string {
+    font-weight: normal;
+  }
+
+  #content-main .historylink {
+    border: 1px solid var(--color-border);
+    border-radius: var(--border-radius-standard);
+  }
+
+  #content-main .historylink:hover {
+    background-color: var(--color-blue) !important;
+  }
```

Comment:
Do you think it makes sense to move the styling to a separate css file or is there anything that speaks against it?

Reply:
It makes sense to extract it to a separate stylesheet. This is a configuration issue, unfortunately I just didn't have time to figure it out.